boards/apollo3/lora_things_plus/src/main.rs

```diff
@@ -378,6 +381,12 @@ unsafe fn setup() -> (
     .finalize(components::air_quality_component_static!());
     CCS811 = Some(ccs811);
 
+    let gnss = I2cNmeaComponent::new(mux_i2c, 0x10)
+        .finalize(components::i2c_nmea_component_static!(apollo3::iom::Iom));
+    let nmea =
+        components::nmea::NmeaComponent::new(board_kernel, capsules_extra::nmea::DRIVER_NUM, gnss)
+            .finalize(components::nmea_component_static!());
```

Comment:
This isn't about the code, but what device is actually providing GNSS? I can't find any documentation about it on the Sparkfun website: https://www.sparkfun.com/products/17506

Reply:
It's an external device.

I originally meant to drop the changes to the board before opening the PR but forgot.

Now thinking about it in context of https://github.com/tock/tock/pull/3873 though, it does make it a lot easier to test and doesn't conflict with anything else

Reply:
I still don't like that we're instantiating arbitrary different peripherals on random boards. Either we should create such a "configuration board" as proposed in #3873 specifically for this external peripheral (which may then also be a great test for our CI infrastructure), or we remove it here. The point behind components is that it's easy to add this to any board, in particular when its a device on I2C or a similar bus.

Reply:
Removed. Hopefully it can be brought back via #3874 or something similar